diff --git a/dist/index.js b/dist/index.js
index 2c40c105731c53938fe7f55c699190abd94ad00a..5e5d9f21a4f49cbe7dc5e6d7ab9b2665630d294d 100644
--- a/dist/index.js
+++ b/dist/index.js
@@ -9,12 +9,7 @@ async function getTokenDataByAddress(mint) {
     if (!mint) {
       throw new Error("Mint address is required");
     }
-    const response = await fetch(`https://tokens.jup.ag/token/${mint}`, {
-      method: "GET",
-      headers: {
-        "Content-Type": "application/json"
-      }
-    });
+    const response = await fetch(`https://tokens.jup.ag/token/${mint}`);
     const token = await response.json();
     return token;
   } catch (error) {
